component/otelcol/exporter/prometheus/prometheus.go

```diff
@@ -35,6 +35,7 @@ type Arguments struct {
 	IncludeScopeLabels bool                 `river:"include_scope_labels,attr,optional"`
 	GCFrequency        time.Duration        `river:"gc_frequency,attr,optional"`
 	ForwardTo          []storage.Appendable `river:"forward_to,attr"`
+	AddMetricSuffixes  bool                 `river:"add_metrics_suffixes,attr,optional"`
```

Comment:
can we change this tag to be `add_metric_suffixes` instead of `add_metrics_suffixes` so we can match what the collector is doing?

Reply:
```suggestion
	AddMetricSuffixes  bool                 `river:"add_metric_suffixes,attr,optional"`
```

Reply:
you're right, I made a typo, will fix it to align with collector 👍